.github/workflows/ci.yml

```diff
@@ -96,8 +100,8 @@ jobs:
         shell: bash
         id: cache-vars
         run: |
-          echo "::set-output name=poetry-venv-dir::$(poetry config virtualenvs.path)"
-          echo "::set-output name=date::$(/bin/date -u "+%m%w%Y")"
+          echo "poetry-venv-dir=$(poetry config virtualenvs.path)" >> $GITHUB_OUTPUT
+          echo "date=$(/bin/date -u "+%m%w%Y")" >> $GITHUB_OUTPUT
```

Comment:
I think we should start using the feature in `action/cache` to cache venv directory instead of doing it outselves

Reply:
See https://github.com/ManimCommunity/manim/pull/3079/commits/1eaa9423a70803dc692db83fb27e6d666aa57d3b